packages/client/components/ReflectionCard/ReflectionCard.tsx

```diff
@@ -254,11 +255,30 @@ const ReflectionCard = (props: Props) => {
     !isComplete &&
     !isDemoRoute() &&
     (isHovering || !isDesktop)
+
+  const firstReflection = reflectionGroups?.[0]?.reflections?.[0]
+  //reflectionCount === reflectionGroups.length showing the weird behavior on reload, that's why calculated like this
+  const primaryCondition = reflectionGroups.find((group) => group.reflections.length > 1)
+  const draggingHappening = (() =>
```

Comment:
-1 Given that you already touched the `firstReflection`, here only needs to determine the state of the first card at this point, not need to iterate through the entire array?

Reply:
-1 Maybe `Array.prototype.some()` might be more meaningful? with the scenario here to return `boolean` value of `is Dragging`? 

> The some() method tests whether at least one element in the array passes the test implemented by the provided function. - [Array.prototype.some() - JavaScript | MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some)

Reply:
> here only needs to determine the state of the first card at this point, not need to iterate through the entire array

Reply:
> show the hint after 30s and repeat every 60s until there is at least one reflection group.
According to the original requirement description, the `remoteDrag` should be ignored, because the drag is an intermediate state that does not end up forming one `reflectionGroup`.

Reply:
So I think the condition of judgment here should be:
```js
const isNotDragging = !reflectionGroups.some(
  (group) =>  group.reflections.some(
    (reflection) => reflection.isViewerDragging)
  )
)
```

Reply:
Ok